crates/proto/src/domain/account.rs

```diff
@@ -345,12 +347,31 @@ impl From<AccountStorageHeader> for proto::account::AccountStorageHeader {
     }
 }
 
+/// Account vault assets
+///
+/// Represents a list of assets, if the number of assets is reasonably small, which
+/// is currently set to 1000 for no particular reason.
```

Comment:
Currently, every asset is 32 bytes - though, we are likely to increase this to 64 bytes in the future. So, 1K assets would be about 64KB of data. It _feels_ a bit high - so, maybe we should reduce this, but I don't have good rationale to what exactly.